source/common/http/conn_manager_impl.cc

```diff
@@ -533,8 +534,12 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
                connection_manager.config_.scopedRouteConfigProvider() == nullptr)),
          "Either routeConfigProvider or scopedRouteConfigProvider should be set in "
          "ConnectionManagerImpl.");
+  for (const AccessLog::InstanceSharedPtr& access_log : connection_manager_.config_.accessLogs()) {
```

Comment:
This pushes the access logs via the config onto the same list as the ones coming via the filter factory, so we avoid having to loop over both this and the list of registered access logs in the FM dtor